tests/test_emu_cli.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import subprocess
+import os
+import pandas as pd
+
+
+@pytest.mark.parametrize(
+    "emu_command, abundance_file, expected_abundancies",
+    [
+        (
+            "./emu abundance --db emu_database --output-dir test_results example/full_length.fa",
+            "test_results/full_length_rel-abundance.tsv",
+            {
+                "Sphingobacterium puteale": 0.5,
+                "Mycobacterium saskatchewanense": 0.25,
+                "Streptococcus sobrinus": 0.25,
+            },
+        ),
+        (
+            "./emu abundance --db emu_database --output-dir test_results --type sr example/short_read_f.fq",
+            "test_results/short_read_f_rel-abundance.tsv",
+            {
+                "Staphylococcus hominis": 1.0,
+            },
+        ),
+        (
+            "./emu abundance --db emu_database --output-dir test_results --type sr example/short_read_f.fq example/short_read_r.fq",
+            "test_results/short_read_f-short_read_r_rel-abundance.tsv",
+            {
+                "Staphylococcus aureus": 1.0 / 3,
+                "Salmonella enterica": 1.0 / 3,
+                "Enterococcus columbae": 1.0 / 3,
+            },
```

Comment:
I'm not completely sure here if it is expected that "Staphylococcus hominis" no longer shows up, and instead we get three equal parts of three other species, although the same forward strand file is used?